build.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright © June Life Inc. 2022. All rights reserved. No part of this file may be
+// distributed, reproduced, or used by entities other than June Life Inc. without express
+// written permission from June Life Inc.
+
+fn main() {
+    build_tools::configure_toolchain();
+}
```

Comment:
This should not be required either (as this crate doesn't itself build any c code). Does something happen which seemed resolved by adding this?

Reply:
Nope, this is a vestige of trying to get things working.  It doesn't need to be in this repo.